pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
+
+- Output is formatted for a 10 columns wide display:
```

Comment:
try : 
```
>printf "header1 header2\nbar foo\n" | column
 header1 header2	bar foo
```
and  
```
>printf "header1 header2\nbar foo\n" | column -c 10
header1 header2
bar foo
```

in this case any number below 31 would not split the line (don't ask me why, it prints 24 characters in output...)